bcipy/gui/mode/RSVPKeyboard.py

```diff
@@ -84,6 +88,28 @@ def _cast_experiment_type(self, experiment_type_string: str) -> None:
             raise ValueError('Register this experiment type or remove button')
 
         return experiment_type
+        
+    def load_items_from_txt(self, event):
+        parameters = load_json_parameters('bcipy/parameters/parameters.json', value_cast=True)
+        data_save_loc = parameters['data_save_loc']
+        #Is this an absolute path?
+        if(os.path.isdir(data_save_loc)):
```

Comment:
This is a good check. Looking at the parameters.json it is not an absolute path. The correct syntax would be `if os.path.isdir(data_save_loc)`, if it's not found raise an error message.